packages/framework/src/Controller/Admin/DefaultController.php

```diff
@@ -14,6 +14,8 @@
 
 class DefaultController extends AdminBaseController
 {
+    protected const PREVIOUS_DAYS_TO_LOAD_STATISTICS_FOR = 7;
```

Comment:
it should be X days. Why the decision was set to just static 7 ?
is that somehow documented, or it is not needed?

Reply:
I chose this because it seemed to be nice interval. This number can be changed from extending class. I would leave the decision about this to BV.